live-examples/js-examples/reflect/reflect-get.html

```diff
@@ -7,7 +7,7 @@
 console.log(Reflect.get(object1, 'x'));
 // expected output: 1
 
-var array1 = ['zero', 'one'];
+let array1 = ['zero', 'one'];
```

Comment:
Here also const will work. @wbamberg what do you think?

Reply:
Agreed.